components/NavBar.tsx

```diff
@@ -79,7 +84,7 @@ function NavBar() {
                     GET STARTED
                   </Link>
                 </MenuItem>
-                <MenuItem>
+                {/* <MenuItem>
```

Comment:
@okhaimie-dev regarding the text shown in the MenuItem that is going to have the `Projects` and `Fundamentals` page, would it be better to actually keep the `Tracks` name on it??

![image](https://github.com/Developer-DAO/academy/assets/2791028/22682a89-8d28-416e-b84b-97af35bc2bc5)

It's like we have only the HomePage and a Get Started nav options, but besides that the get started is the only page we have, it cointains all of the D_D Academy tracks aka projects, and the fundamentals. 
Having SEO and user guidence in mind, perhaps it would be better to name it as "Tracks", or with another name.

cc @wolovim @kempsterrrr asking thoughts on this :pray: 

in case this will changed it's a small change,

Thak you Okhai for your help here! :raised_hands: :muscle:

Reply:
decided to leave it as is :100: